### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by MsLGXC**
remove all periods in schinese and some small changes
remove residual periods form other lang
unexpectedly discovered an invalid french entry and added ""

### CODE REVIEW

File: achievements/224260_loc_tchinese.vdf [hunk 1]
```diff
@@ -1,12 +1,12 @@
-"lang"
+"lang""
```
Comment (felis-catus) [inline achievements/224260_loc_tchinese.vdf:1]: Duplicate `"`, this will break parsing.

File: news/release/1133_localization_schinese.xml
Comment (felis-catus) [inline news/release/1133_localization_schinese.xml:1]: It appears that file encoding got changed. Can you ensure it remains UTF-8? If you're using Notepad++, you can convert by selecting Encoding -> UTF-8.

File: news/release/1134_localization_schinese.xml
Comment (felis-catus) [inline news/release/1134_localization_schinese.xml:1]: Same comment regarding file encoding, it should remain UTF-8.

**Review by felis-catus — CHANGES_REQUESTED** [review]
Found some issues that should be addressed.

File: news/release/1133_localization_schinese.xml
Comment (MsLGXC) [reply news/release/1133_localization_schinese.xml:1]: No, I haven't touched the news. These files were pulled locally from the latest updates, and I don't know why GitHub thinks they have been modified

File: news/release/1134_localization_schinese.xml
Comment (MsLGXC) [reply news/release/1134_localization_schinese.xml:1]: I display it as utf8 on the local file and I haven't even opened them?

File: news/release/1133_localization_schinese.xml
Comment (MsLGXC) [reply news/release/1133_localization_schinese.xml:1]: Both vscode and notebook display it as utf8

Comment (felis-catus) [reply news/release/1133_localization_schinese.xml:1]: Oh, it also happened to me on pull, seems to be caused by the existing file, not sure why... we can close this issue.

**Review by felis-catus — APPROVED** [review]
Looks good to me.

**Pull request merged**